.github/workflows/actions.yml

```diff
@@ -7,14 +7,36 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    # A matrix proves the supported range of Go versions work. This must always
+    # include the floor Go version policy of Mosn and the current Go version.
+    # Mosn binaries are built with a specific version inside this range. Mosn's
+    # floor Go version is two behind current. For example, if Go supports 1.20,
+    # Mosn supports 1.18, 1.19 and 1.20.
+    #
+    # A floor version is required to ensure Mosn can receive security patches.
+    # Without one, dependencies become locked to an old version of Go, which
+    # itself receives no security patch updates.
+    #
+    # Note: Mosn might work with a Go version below its supported floor, but
+    # users must not depend on this. Mosn's library dependencies are free to
+    # features in it supported floor Go version at any time.
+    #
+    # Note: There are no unit tests in this project, so we run a matrix on
+    # build, not test.
+    strategy:
+      matrix:
+        go-version:
+          - "1.19"  # Current Go version
+          - "1.17"  # Floor Go version of Mosn == current - 2
```

Comment:
missing 1.18?
![image](https://user-images.githubusercontent.com/52443884/199176983-cc56708f-7cc2-47ae-b712-f0eba763bea2.png)

Reply:
In our project we only test the ends because it is unlikely 1.18 would fail if 1.17 and 1.19 succeed. It is a little less CI burning energy. That said, If you feel you want to also see 1.18 no problem.

Reply:
I'll add it. and also make a note about mosn binary being built with Go's floor